pkg/conference/participant/tracker.go

```diff
@@ -3,26 +3,34 @@ package participant
 import (
 	"fmt"
 
-	"github.com/matrix-org/waterfall/pkg/conference/subscription"
+	pub "github.com/matrix-org/waterfall/pkg/conference/track"
```

Comment:
Yeah, it is, I think the reason I added it is that without the alias it would be `track.XXX` while `track` is also a very common variable name in the code, so it clashes very often and gives a confusing error. Whereas with `pub` it's just `pub.PublishedTrack` which does not look too bad.

But I could remove the alias and rather rename the variables from `track` to `trackRemote`, `publishedTrack`, etc. What would you prefer?

Reply:
Possibly, yes, this seems like a symptom of having a namespace clash in the first place. It might be better to try & avoid the namespace clash by not calling things `track`, as you suggest.

Reply:
Yeah, agree. Originally I used the `pub` alias for `track` package since the `track` is quite a simple and a likely name that a variable can have, so the conflicts are not rare. Alternatively one could rename the package from `track` to something like `published_track` (which is more descriptive for a module), but such package names is almost considered an antipattern for Go and since regardless of the path, only the latest package name is used as a "default" alias, clashes are not very uncommon in Go unfortunately (while using aliases is also not supper common).

Anyway, I've removed the `pub` alias (in fairness "pub" is not a very descriptive alias either) and renamed the variables inside a function. Ptal 🙂

Reply:
Thanks!